src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Test/Index.cshtml

```diff
@@ -32,7 +37,7 @@
                     }
                     else
                     {
-                        if (Model.SponsorLogoUrl != null && Model.IsAdvent)
+                        if (Model.SponsorLogoUrl != null && isAdvent)
```

Comment:
it's ok, but we can change the order - first check isAdvent and then check model